inc/cleantalk-settings.php

```diff
@@ -2569,12 +2571,36 @@ function apbct_settings__sanitize__exclusions($exclusions, $regexp = false)
         } else {
             $exclusions = explode(',', $exclusions);
         }
-        foreach ( $exclusions as $exclusion ) {
-            $sanitized_exclusion = trim($exclusion, " \t\n\r\0\x0B/\/");
+        //Take first 20 exclusions entities
+        if (array_count_values($exclusions) > 20) {
```

Comment:
may be count()?

Reply:
I'm agree with the opinion to use count() too.